x-pack/legacy/plugins/alerting_ui/np_ready/public/application/app.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Switch, Route, Redirect } from 'react-router-dom';
+import { BASE_PATH, Section, DEFAULT_SECTION } from './constants';
+import { AlertsUIHome } from './home';
+
+class ShareRouter extends Component {
+  static contextTypes = {
+    router: PropTypes.shape({
+      history: PropTypes.shape({
+        push: PropTypes.func.isRequired,
+        createHref: PropTypes.func.isRequired,
+      }).isRequired,
+    }).isRequired,
+  };
+  constructor(props: any, context?: any) {
+    super(props, context);
+  }
+
+  render() {
+    return this.props.children;
+  }
+}
+
+export const App = (api: any) => {
```

Comment:
`api` is not used here

Reply:
Fixed in https://github.com/elastic/kibana/pull/47843/commits/85578a4fc084818300dbc9fe472dab2050dce9a5.